frontend/src/modules/vehicles/SellVehicle.vue

```diff
@@ -165,7 +177,6 @@ export default {
             this.showMessage("The sale request has been updated", "success");
             this.setVehicleModal(false);
             this.setSaleStatus(true, response.data.payload);
```

Comment:
Lines 177 - 179 are almost identical to lines 150-152. Should we refactor? Since it's only used twice it might not be worth it to create another function

Reply:
I don't think a refactor is needed since these will be the only 2 instances where the code will be that similar

Reply:
Ok yeah that makes sense